lib/galaxy_test/base/populators.py

```diff
@@ -2243,7 +2243,7 @@ def download_workflow(
         if history_id is not None:
             params["history_id"] = history_id
         if instance is not None:
-            params["instance"] = instance
+            params["instance"] = str(instance).lower()
```

Comment:
mypy infers the the type based on the first element that is being set.
```suggestion
            params["instance"] = instance
```

Reply:
then on line 2240 do `params: Dict[str, Any] = {}` (and make sure those are imported)